Components/Buttons/buttons.css

```diff
@@ -0,0 +1,183 @@
+@import url("../../default.css");
+
+/* buttons */
+.btn {
+  display: inline-flex;
+  justify-content: center;
+  align-items: center;
+  padding: 0.5rem;
+  cursor: pointer;
+  border-radius: 5px;
+  transition: all 0.5s;
+  font-size: 1.1rem;
+  min-width: 7rem;
+  border: none;
+}
+
+/* link buttons */
+.btn-link {
+  text-decoration: none;
+  text-align: center;
+  min-width: fit-content;
+  min-height: fit-content;
+}
+
+/* normal buttons */
+
+.btn-primary {
+  background-color: var(--primary-color);
+}
+.btn-primary:hover {
+  background: var(--primary-color-shade);
+}
+
+.btn-secondary {
+  background-color: var(--secondary-color);
+}
+.btn-secondary:hover {
+  background: var(--secondary-color-shade);
+}
+
+.btn-ternary {
+  background-color: var(--ternary-color);
+}
+.btn-ternary:hover {
+  background: var(--ternary-color-shade);
+}
+
+.btn-success {
+  background-color: var(--success-color);
+}
+.btn-success:hover {
+  background: var(--success-color-shade);
+}
+
+.btn-warning {
+  background-color: var(--warning-color);
+}
+.btn-warning:hover {
+  background: var(--warning-color-shade);
+}
+
+.btn-danger {
+  background-color: var(--danger-color);
+}
+.btn-danger:hover {
+  background: var(--danger-color-shade);
+}
+
+.btn-dark {
+  background-color: var(--dark-color);
+  color: var(--light-color);
+}
+.btn-dark:hover {
+  background: var(--dark-color-shade);
+}
+
+/* outline buttons */
+/* button-outline */
+.btn-primary-outline {
+  background-color: var(--light-color);
+  border: 2px solid var(--primary-color);
+  color: var(--primary-color);
+}
+.btn-primary-outline:hover {
+  background: var(--primary-color);
+  color: var(--light-color);
+}
+
+.btn-secondary-outline {
+  background-color: var(--light-color);
+  border: 2px solid var(--secondary-color);
+  color: var(--secondary-color);
+}
+.btn-secondary-outline:hover {
+  background: var(--secondary-color);
+  color: var(--light-color);
+}
+
```

Comment:
small suggestion
if you want your outline btn to have the same color as the parent then you can use 
background: inherit

Reply:
Thanks. I will keep it in mind